src/main/java/org/jitsi/videobridge/Conference.java

```diff
@@ -1673,4 +1682,22 @@ public String getName()
     {
         return name;
     }
+
+    public void addPushEvent(JSONObject event)
+    {
+      System.out.println("CONF ADDING PUSH EVENT");
+      try {
+        pushEventQueue.put(event);
```

Comment:
yeah i agree...a max size that returns some value on failure (so the caller can potentially try again) seems good.  but at a higher level, this queue (especially in this location) was really just a hack.  doesn't the conference already have a mechanism for pushing events out on the xmpp channel?  i'd think there should be some code we can re-use from there